### ISSUE TITLE
NIX Sigma 3.0rc2 - showing wrong amount of ghosted Nix

### ISSUE THREAD

**Issue by Knightofthebooks** [NixPlatform/NixCore#36]
Showing probably wrong amount of ghosted Nix. Tried to unghost complete amount of ghosted Nix --> Warning appears.

Debug log:
[debug nix 4.log](https://github.com/NixPlatform/NixCore/files/3192537/debug.nix.4.log)

![ghost vault incorrect](https://user-images.githubusercontent.com/50663678/57946537-ce40c400-78dc-11e9-863a-c5d3d900ba99.jpg)

**Comment by Knightofthebooks** [comment]
To solve the problem mentioned above I deleted chainstate and blocks folder. After reindexing the wallet a wrong amount of Nix is displayed. 
Closing again the wallet after reindexing and restarting it the correct amount of Nix appears. Which means that the amount of ghosted Nix is 100 and not 156 (see screenshot above).

Starting wallet after deleting chainstate and blocks folder:
![after reindexing 2](https://user-images.githubusercontent.com/50663678/57964412-7c785800-7935-11e9-913e-589fb2548cc6.jpg)

After starting wallet for second time correct amount of Nix is displayed:
![after restarting](https://user-images.githubusercontent.com/50663678/57964430-b0537d80-7935-11e9-82b2-5df187a0b6fb.jpg)
